examples/mnist-classification/src/mnist_classification/main.clj

```diff
@@ -14,7 +14,8 @@
 
 (defn -main
   [& args]
+  (println "Welcome! Please wait while we compile some Clojure code...")
   (let [argmap (parse-opts args cli-options)]
-    (require 'suite-classification.core)
-    ((resolve 'suite-classification.core/train-forever-uberjar)
+    (require 'mnist-classification.core)
```

Comment:
Not sure, this was already there. I think this hack makes some REPLs start up a bit faster.